wfe/web-front-end.go

```diff
@@ -708,7 +708,11 @@ func (wfe *WebFrontEndImpl) Certificate(response http.ResponseWriter, request *h
 
 		cert, err := wfe.SA.GetCertificateByShortSerial(serial)
 		if err != nil {
-			wfe.sendError(response, "Not found", err, http.StatusNotFound)
+			if strings.HasPrefix(err.Error(), "gorp: multiple rows returned") {
```

Comment:
Gorp is already doing a multiple-row select on the call to [SelectOne](https://github.com/go-gorp/gorp/blob/master/gorp.go#L1537) and, annoyingly, is not returning a const object.

I think this is a flaw with Gorp, but rather than code around it, let's keep the code as it is and let Gorp improve the situation. I'm going to file a bug upstream and will note it here.